tests/perfsys.ts

```diff
@@ -20,29 +20,9 @@ module perftest {
     export var getCurrentDirectory = ts.sys.getCurrentDirectory;
     var exit = ts.sys.exit;
 
-    var args = ts.sys.args;

```

Comment:
Perhaps @vladima can weigh in on this, but was this specifically something that was necessary to simplify the *Jakefile*?